stdlib/3/http/__init__.pyi

```diff
@@ -0,0 +1,63 @@
+import sys
+
+from enum import IntEnum
+
+if sys.version_info >= (3, 5):
+    class HTTPStatus(IntEnum):
+        CONTINUE = ...  # type: object
```

Comment:
I propose to add
```
        def __init__(self, *a) -> None:
            self.phrase = ...  # type: str
            self.description = ...  # type: str
```
here so that users will find that HTTPStatus.phrase has type str (as it does at runtime).